tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -42,59 +43,65 @@ def generate_data(self, frames: list[dict[str, str | bool]], platform: str = "")
         test_data = {"platform": platform or self.platform, "stacktrace": {"frames": frames}}
         return self.store_event(data=test_data, project_id=self.project.id).data
 
+    def assert_slo_metric(self, mock_record, outcome=EventLifecycleOutcome.FAILURE, error_msg=None):
+        (events,) = (call for call in mock_record.mock_calls if call.args[0] == outcome)
+        if error_msg:
+            assert events.args[1] == error_msg
```

Comment:
I think Raj has some shared metrics utils to assert errors [here](https://github.com/getsentry/sentry/blob/d66e44f3a830007d07a83c1b248f7c752ab8c370/tests/sentry/integrations/utils/test_assert_metrics.py#L18). Could we use those instead?

Reply:
done, thanks!